### ISSUE TITLE
Report Tasks aus dem Weekly 9

### ISSUE THREAD

**Issue by gabrieltorresgamez** [AdversarialAttacks/report#2]
- [x] Krebsklassenanalyse: Welche Krebsarten haben wir und kann man da zwischen Bösartig und nicht Bösartig unterscheiden.
- [x] Orthographiecheck: Wir schreiben die gleichen Sachen recht unterschiedlich (z.B. BinaryCrossEntropy | Binary Cross Entropy | Binary-Cross-Entropy. Hier müssen wir einheitlicher schreiben! Eventuell ein Orthographietool für LaTeX suchen.
- [x] Uns fehlt die Datenexplorierung (Nicht stark auf Testdaten schauen!).
- [x] Wir sollten unsere Technische Umgebung sowie genutzte Libraries etc. dokumentieren.
- [x] Wir sollten eine schematische Skizze erstellen, welche unser ganzes Projekt beschreibt. Diese sollten wir am besten vor dem Preprocessing erwähnen.
- [x] Algorithmen, Preprocessing etc. sollte mehr theoretisch als praktisch dokumentiert sein (Library-unabhängig).
- [x] Baselinetraining Ergebnisse sollten im Kapitel 5 sein.
- [x] Baseline Modelle sollten wir umbennen, da der Begriff "Baseline" in anderen Kontexten genutzt wird.
- [x] Bei der Fooling Rate sollten wir für den booleanischen Begriff die Indikatorfunktion (\mathcal) nutzen.
- [x] Die Roundfunktion muss nicht mathematisch definiert werden, sie kann im Prosa beschrieben werden.
- [x] Speziell betonen, dass wir die Loss Funktion so optimiert haben um das Optimierungsproblem vom Paper zu lösen
- [x] Bei der UAP Loss Function beschreiben, welche Parameter genau optimiert werden (delta_v).
- [x] Bei der Loss funktion beschreiben, wie sich yhat_adv Zusammensetzt (img + v + delta_v).

**Comment by 7ben18** [comment]
Krebsklassenanalyse - MRI Datensastz: https://sartajbhuvaji.github.io/Brain-Tumor-Classification-Website/

Informationen zu den Gehirntumor Arten von [USZ](https://www.usz.ch/krankheit/hirntumore-hirnmetastasen/#:~:text=WHO%2DGrad%20I%3A%20Es%20ist,Tendenz%2C%20nach%20einer%20Behandlung%20wiederzukehren.&text=WHO%2DGrad%20IV%3A%20Der%20Tumor,und%20besitzt%20eine%20ung%C3%BCnstige%20Prognose.): 

Wichtiges Zitat: "Manche sind auch zunächst gutartig und gehen dann in eine bösartige Variante über." 

Hirntumore werden in 4 WHO-Graden unterteilt:
WHO-Grad I: Es ist ein gutartiger Tumor, der langsam wächst und eine günstige Prognose besitzt.
WHO-Grad II: Der Tumor ist nicht mehr gutartig und hat die Tendenz, nach einer Behandlung wiederzukehren.
WHO-Grad III: Der Tumor ist bösartig
WHO-Grad IV: Der Tumor ist sehr bösartig, wächst schnell und besitzt eine ungünstige Prognose.

Zu den Gehirntumoren in unserem Datensatz:GLIOMA TUMOR
[Gliome](https://www.usz.ch/krankheit/glioblastom/):  Die bösartigste Form einer Gewebeentartung im Gehirn. Wird in WHO-Grad IV eingeteilt. 

[Malignent](https://de.wikipedia.org/wiki/Malignit%C3%A4t): Bedeutet Bösartiger Gehirntumor. utartige (benigne) oder bösartige (maligne)


[Pituary](https://neurochirurgie.insel.ch/erkrankungen-spezialgebiete/hirntumoren/hypophysenadenom#:~:text=Hypophysenadenome%20sind%20gutartige%20Tumoren%20der,Hypophysenadenome%20verursachen%20hormonelle%20St%C3%B6rungen.): Heist Hypohysenadenome und treten 10% von den Gehirntumoren auf und sind gutartige Tumore.
